f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -500,9 +500,78 @@ def __init__(self, driver):
     def _create_payload(self, member, service):
         return self.driver.service_adapter.get_member(service)
 
+    def _merge_members(self, lbaas_members, bigip_members, service):
+        members_payload = []
+        name_list = []
+
+        """ old members from bigip """
+        for item in bigip_members:
+            fqdn = item.fqdn
+            content = item.to_dict()
+            del content['_meta_data']
+            del content['fqdn']
+            """ after to_dict the content's fqdn becomes a list somehow"""
+            content['fqdn'] = fqdn
+
+            if content['session'] == 'monitor-enabled':
+                del content['session']
+                del content['state']
+            elif content['session'] == 'user-disabled':
+                if content['state'] != 'user-down':
+                    del content['state']
+            else:
+                LOG.debug("Should not reach here %s %s",
```

Comment:
there might be some other case, for instance, I used to see session property value is 'monitor-enabled'...

Reply:
```json
{
    "kind": "tm:ltm:pool:members:memberscollectionstate",
    "selfLink": "https://localhost/mgmt/tm/ltm/pool/~Project_5414450ea0474a2aadd9ec4640d8964d~Project_b653c48e-94e8-4106-bbdf-aa53c9b425f6/members?ver=13.1.1.4",
    "items": [
        {
            "kind": "tm:ltm:pool:members:membersstate",
            "name": "203.0.102.114%1:80",
            "partition": "Project_5414450ea0474a2aadd9ec4640d8964d",
            "fullPath": "/Project_5414450ea0474a2aadd9ec4640d8964d/203.0.102.114%1:80",
            "generation": 1581,
            "selfLink": "https://localhost/mgmt/tm/ltm/pool/~Project_5414450ea0474a2aadd9ec4640d8964d~Project_b653c48e-94e8-4106-bbdf-aa53c9b425f6/members/~Project_5414450ea0474a2aadd9ec4640d8964d~203.0.102.114%251:80?ver=13.1.1.4",
            "address": "203.0.102.114%1",
            "connectionLimit": 0,
            "dynamicRatio": 1,
            "ephemeral": "false",
            "fqdn": {
                "autopopulate": "disabled"
            },
            "inheritProfile": "enabled",
            "logging": "disabled",
            "monitor": "default",
            "priorityGroup": 0,
            "rateLimit": "disabled",
            "ratio": 1,
            "session": "monitor-enabled",
            "state": "down"
        }
    ]
}
```